src/Features/CSharp/Portable/CodeRefactorings/UseRecursivePatterns/UseRecursivePatternsCodeRefactoringProvider.cs

```diff
@@ -600,7 +600,7 @@ static bool CanConvertToSubpattern(IdentifierNameSyntax name, SemanticModel mode
 
         // Process all nodes to refactor in reverse to ensure nested nodes
         // are processed before the outer nodes to refactor.
-        foreach (var originalNode in nodes.Reverse())
+        foreach (var originalNode in Enumerable.Reverse(nodes))
```

Comment:
>Enumerable.Reverse(nodes) [](http://example.com/codeflow?start=37&length=25)

Why was this change needed? Are we now binding `nodes.Reverse()` to `MemoryExtensions.Reverse<T>(Span<T>)`? And if so, is that an issue? (It looks like the array that is created from `var nodes = ... ToArray();` should not include array variance.)

Reply:
> Are we now binding `nodes.Reverse()` to `MemoryExtensions.Reverse<T>(Span<T>)`?

Yes.

> And if so, is that an issue?

Yes, because `MemoryExtensions.Reverse` returns `void`.